examples/agent_client/examples/agent_client.rs

```diff
@@ -158,42 +133,28 @@ fn load_configs() -> Result<(AgentConfig, KafkaConfig), String> {
         timeout_ms: read_var("AGENT_TIMEOUT_MS").unwrap().parse().unwrap(),
     };
 
-    let cfg_kafka = KafkaConfig {
-        brokers: read_var("KAFKA_BROKERS")?,
-        group_id: read_var("KAFKA_GROUP_ID")?,
-        certification_topic: read_var("KAFKA_TOPIC")?,
-        fetch_wait_max_ms: read_var("KAFKA_FETCH_WAIT_MAX_MS")?.parse().map_err(|e: ParseIntError| e.to_string())?,
-        message_timeout_ms: read_var("KAFKA_MESSAGE_TIMEOUT_MS")?.parse().map_err(|e: ParseIntError| e.to_string())?,
-        enqueue_timeout_ms: read_var("KAFKA_ENQUEUE_TIMEOUT_MS")?.parse().map_err(|e: ParseIntError| e.to_string())?,
-        log_level: get_kafka_log_level_from_env()?,
-        talos_type: TalosType::External,
-        sasl_mechanisms: read_var_optional("KAFKA_SASL_MECHANISMS")?,
-        username: read_var_optional("KAFKA_USERNAME")?,
-        password: read_var_optional("KAFKA_PASSWORD")?,
-    };
+    let mut cfg_kafka = KafkaConfig::from_env(Some("AGENT"));
+    let more_producer_values = [
+        ("message.timeout.ms".to_string(), "15000".to_string()),
```

Comment:
Could not find a way to avoid .to_string(). Original interface was `HashMap<&static str, ...>` which I could not keep due to compiler errors.